source/blog/openshift-openId-integration-with-engine-sso.html.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Configuring Kibana/Elastic Search/Open Shift backend to authenticate with engine sso
+author: rnori
+tags: community, documentation, howto, openshift, ovirt, sso
+date: 2017-04-04 11:40:00 CET
+comments: true
+published: true
+---
+
+Configuring Kibana/Elastic Search/Open Shift backend to authenticate with engine sso.
+
+A step by step instruction on how to configure open shift to authenticate with engine sso.
+
+READMORE
+
+## Background
+
+Kibana/Elastic Search/OpenShift needs to be setup to authenticate using engine so that the users on engine can access Kibana using engine sso to authenticate. 
+
+## Prerequisites
+
+* A fully working and configured oVirt engine instance
+* A fully working and configured instance of Kibana/Elastic Search and Open Shift
+
+## Installing Kibana/Elastic Search and Open Shift backend
+
+Install Kibana/Elastic Search/OpenShift on CentOS7 or RHEL 7.3 as described in http://git.engineering.redhat.com/git/users/rmeggins/test-openshift-heat.git/tree/README-mux.md
+
+## Installing oVirt Engine
+
+Setup engine on a separate host ovirtengine.redhat.com as described in https://www.ovirt.org/download/
+
+## Setting up engine certtificate on open shift machine
+
+Get the ovirt-engine CA as described here https://www.ovirt.org/documentation/how-to/guest-console/connect-to-spice-console-without-portal/
+```sh
+scp root@${OVIRT}:/etc/pki/ovirt-engine/ca.pem ${CA_FILE}
+```
+Add the certificate to system wide trusted certificates
+Copy certificate to /etc/pki/ca-trust/source/anchors/ and run update-ca-trust
+
+## Register a new sso client on ovirty-engine host
+
+Run the new client registration tool ovirt-register-sso-client introduced by patch https://gerrit.ovirt.org/#/c/74212/ to register the new sso client. The tool will prompt the user to enter the client id, location of the client certificate and the callback url prefix. Make note of the client id and client secret generated by the tool. This information is needed in the master-config.yaml file.
```

Comment:
"This information is needed..." - please describe why it is needed so that the person has an idea why he is asked for the information. He does not care where it is stored.